config/sync/editor.editor.stanford_html.yml

```diff
@@ -56,7 +56,7 @@ settings:
             - A11ychecker
   plugins:
     stylescombo:
-      styles: "a.su-button|Button\r\na.su-button--big|Big Button\r\na.su-button--secondary|Secondary Button\r\na.su-link--action|Action Link\r\np.plain-text|Normal\r\np.su-intro-text|Intro Text\r\np.su-font-splash|Splash Font\r\np.su-quote-text|Quote Text\r\np.su-drop-cap|Drop Cap First Letter\r\np.su-related-text|Card Text\r\np.su-callout-text|Callout Text\r\np.su-subheading|Sub Title\r\nol.cs-number--big|Number List"
+      styles: "a.su-button|Button\r\na.su-button--big|Big Button\r\na.su-button--secondary|Secondary Button\r\na.su-link--action|Action Link\r\np.plain-text|Normal\r\np.su-intro-text|Intro Text\r\np.su-font-splash|Splash Font\r\np.su-quote-text|Quote Text\r\np.su-drop-cap|Drop Cap First Letter\r\np.su-related-text|Card Text\r\np.su-callout-text|Callout Text\r\np.su-subheading|Sub Title\r\nol.cs-number--big|Number List\r\ntbody.cs-table--span3|1/4 Table"
```

Comment:
When i first tried this, it added an empty `tbody` tag into the table. I think we should move this to the `table` element. then it'll apply the 15% to the header row and the regular rows.

Reply:
@rebeccahongsf Don't forget to change this to `table`